Cargo.toml

```diff
@@ -10,13 +10,15 @@ exclude = ["test_fixture"]
 
 [dependencies]
 bytes = "0.4.12"
-futures = "0.1.29"
+futures = "0.3.5"
 iovec = "0.1.4"
-libc = "0.2.65"
-mio = "0.6.19"
-nix = "0.15.0"
-tokio = "0.1.22"
-vsock = "0.1.4"
+libc = "0.2.71"
+mio = "0.6.20"
+nix = "0.17.0"
+tokio = { version = "0.2", features = ["io-driver", "stream", "macros", "rt-threaded", ] }
+vsock = { git = "https://github.com/fsyncd/vsock-rs", branch = "master" }
```

Comment:
This should be updated after `vsock-rs` published on crates.io.

Reply:
I've published the updated crate for `vsock-rs` so can be changed now.

Reply:
> I've published the updated crate for `vsock-rs` so can be changed now.

done.